recipes/bison/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import glob
+import os
+import shutil
+
+
+class ConanFileDefault(ConanFile):
+    name = "bison"
+    url = "https://github.com/conan-io/conan-center-index   "
+    homepage = "https://www.gnu.org/software/bison/"
+    description = "Bison is a general-purpose parser generator"
+    topics = ("conan", "bison", "parser")
+    license = "GPL-3.0-or-later"
+    exports_sources = ["patches/*.patch"]
+    _source_subfolder = "source_subfolder"
+    requires = ("m4/1.4.18", "flex/2.6.4")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "bison-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        self._build_configure()
+
+    def _build_configure(self):
+        true = tools.which("true") or "/bin/true"
```

Comment:
What is this line doing? 😄

Reply:
It is used in the line below it as an *alternative implementation* of help2man and makeinfo.

Reply:
I removed these lines and created a rudimentary python implementation.